README.md

```diff
@@ -27,36 +27,38 @@ will automatically see the updates.
 ## The website
 
 ### 1. Build root files
+
 To create some root files from `archives.yaml`:
-- `archive.html` - list of advent calendars
+
+- `archives.html` - list of advent calendars
 - `archives-AZ.html` - list of modules
 - `archives-Yd.html` - list of articles
 
 Execute the following script:
+
 ```bash
 $ perl mkarchives .
 ```
 
 ### 2. Build all (recent) advent calendars
+
 ```bash
-$ sudo cpanm WWW::AdventCalendar
+cpm install -g WWW::AdventCalendar App::HTTPThis
 ```
 
 Then build all recent calendars
+
 ```bash
-$ for year in 2011 2012 2013 2014 2015 2016 2017 2018 2019 2020 2021 2022; do cd $year && advcal -c advent.ini -o `pwd` && cd ..; done
+$ for year in $(seq 2011 2023); do cd $year && advcal -c advent.ini -o `pwd` && cd ..; done
```

Comment:
nice

Reply:
😄 I picked `seq` up from @util's TPRC talk "Command-line Filters - Time to Shine".